[2023-03-01 12:59:59,840] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-26T20:05:00+00:00 [queued]>
[2023-03-01 13:00:00,032] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-26T20:05:00+00:00 [queued]>
[2023-03-01 13:00:00,036] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:00:00,037] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:00:00,042] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:00:00,254] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-26T20:05:00+00:00
[2023-03-01 13:00:00,306] {standard_task_runner.py:52} INFO - Started process 17035 to run task
[2023-03-01 13:00:00,381] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-11-26T20:05:00+00:00', '--job-id', '4051', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmpu4yt7w1o', '--error-file', '/tmp/tmpshyb0g1x']
[2023-03-01 13:00:00,405] {standard_task_runner.py:77} INFO - Job 4051: Subtask get_name
[2023-03-01 13:00:01,064] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-11-26T20:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:00:01,656] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-11-26T20:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-26T20:05:00+00:00
[2023-03-01 13:00:01,883] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:00:02,016] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221126T200500, start_date=20230301T125959, end_date=20230301T130001
[2023-03-01 13:00:02,682] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:00:02,839] {local_task_job.py:146} INFO - Task exited with return code 0
